packages/compass-import-export/src/import/import-csv.ts

```diff
@@ -85,22 +85,14 @@ export async function importCSV({
         }
 
         parsedHeader = {};
-        for (const [index, name] of headerFields.entries()) {
-          try {
-            parsedHeader[name] = parseCSVHeaderName(name);
-          } catch (err: unknown) {
-            // rethrow with the row and column indexes appended to aid debugging
-            (err as Error).message = `${
-              (err as Error).message
-            } [Col ${index}][Row 0]`;
```

Comment:
It could explicitly throw before and now it can't. Unless there's some NPE type programmer error but in that case I think we don't need special handling. Or rather: I can't unit test it anymore.